src/ripple/app/ledger/impl/LedgerMaster.cpp

```diff
@@ -224,44 +226,45 @@ class LedgerMasterImp
         return true;
     }
 
-    int getPublishedLedgerAge () override
+    std::chrono::seconds
+    getPublishedLedgerAge() override
     {
-        std::uint32_t pubClose = mPubLedgerClose.load();
-        if (!pubClose)
+        std::chrono::seconds pubClose{mPubLedgerClose.load()};
+        if (pubClose == 0s)
         {
             JLOG (m_journal.debug) << "No published ledger";
-            return 999999;
+            return weeks{2};
         }
 
-        // VFALCO int widening?
-        auto ret = app_.timeKeeper().closeTime().time_since_epoch().count();
-        ret -= static_cast<std::int64_t> (pubClose);
-        ret = (ret > 0) ? ret : 0;
+        std::chrono::seconds ret = app_.timeKeeper().closeTime().time_since_epoch();
+        ret -= pubClose;
+        ret = (ret > 0s) ? ret : 0s;
 
-        JLOG (m_journal.trace) << "Published ledger age is " << ret;
-        return static_cast<int> (ret);
+        JLOG (m_journal.trace) << "Published ledger age is " << ret.count();
+        return ret;
     }
 
-    int getValidatedLedgerAge () override
+    std::chrono::seconds
+    getValidatedLedgerAge() override
     {
-        std::uint32_t valClose = mValidLedgerSign.load();
-        if (!valClose)
+        std::chrono::seconds valClose{mValidLedgerSign.load()};
+        if (valClose == 0s)
         {
             JLOG (m_journal.debug) << "No validated ledger";
-            return 999999;
+            return weeks{2};
```

Comment:
Understood.  I reviewed all the uses and they are all testing if they are less than something on the order of minutes.  999999s is about 11.6 days.  So this change was only for readability purposes (and subjective).